_sass/blocks/_revenues_subpage_involved.scss

```diff
@@ -0,0 +1,51 @@
+// Revenues Subpage Involved Section
+//
+// This block is where we post videos and social
+// media links on how to become involved
+//
+// Styleguide blocks.revenues_subpage_involved
+
+.revenues_subpage-involved {
+  background-color: $white;
+  padding-top: $base-padding-large;
+  padding-bottom: $base-padding-large;
+
+  h2 {
+    color: $light-black;
+    font-weight: $weight-light;
+  }
+
+  h3 {
+    padding-bottom: 5px;
+    margin-bottom: 8px;
+  }
+
+  .fa-stack {
+    line-height: 2.1em;
+  }
+
+
+}
+
+.revenues_subpage-involved_option {
+  @include respond-to(small-up) {
+      @include span-columns(4 of 8);
+      @include omega(2n);
+  }
+
+  @include respond-to(medium-up) {
+    margin-top: $base-padding-large;
+    @include omega-reset(2n);
+    @include span-columns(3 of 12);
+    @include omega(4n);
+  }
+}
+.revenues_subpage-involved_video {
```

Comment:
Selector `revenues_subpage-involved_video` should be written in lowercase with hyphens